python/minimasar/test/test_ops_postgres.py

```diff
@@ -0,0 +1,493 @@
+
+
+import unittest
+
+from p4p.wrapper import Value
+from p4p.nt import NTTable
+
+from minimasar.db_postgres import connect
+from minimasar.ops_postgres import Service
+
+from p4p.rpc import RemoteError
+
+import numpy
+from numpy.testing import assert_equal
+from minimasar import ops_base, db_postgres
+
+#import dbsettings_postgres
+
+import testing.postgresql
+from sqlalchemy import create_engine
```

Comment:
It looks like you did play with one of the (many) ORMs.  Why did you decide not to use sqlalchemy, nor one of the others?

Reply:
No particular reason, sqlalchemy is only referred to in unit test as testing.postgresql depends on it.